docs/platforms/powershell/data-management/data-collected.mdx

```diff
@@ -0,0 +1,79 @@
+
+## Information About Logged-in User
+
+By default, the Sentry SDK doesn't send any information about the logged-in user, such as email address, user ID, or username. Even if enabled, the type of logged-in user information you'll be able to send depends on the integrations you enable in Sentry's SDK. Most integrations won't send any user information. Some will only set the user ID, but there are a few that will set the user ID, username, and email address.
+
+To start sending logged-in user information, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+## Users' IP Address
+
+By default, the Sentry SDK doesn't send the user's IP address. Even if enabled, whether you're able to send the user's IP address or not, will depend on the integrations you enable in Sentry's SDK. Most integrations won't set the user's IP address at all.
+
+To enable sending the user's IP address, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+## Request URL
+
+The full request URL of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Request Query String
+
+The full request query string of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Request Body
+
+The request body of incoming HTTP requests can be sent to Sentry. Whether it's sent or not, depends on the type and size of request body as described below:
+
+- **The type of the request body:**
+  - JSON and form bodies are sent
+  - Raw request bodies are always removed
+  - Uploaded files in the request bodies are never sent to Sentry
+- **The size of the request body:** There's a ["max_request_body_size"](/platforms/python/configuration/options/#max-request-body-size) option that's set to `medium` by default. This means that larger request bodies aren't sent to Sentry.
+
+If you want to prevent bodies from being sent to Sentry altogether, set `max_request_body_size` to `"never"`.
+
+## Source Context
+
+When an unhandled exception is sent to Sentry, a snapshot of the source code surrounding the line where the error originates is sent with it.
+
+To opt out of sending this source context to Sentry, set `include_source_context` to `False`.
+
+## Local Variables In Stack Trace
+
+When unhandled errors and exceptions are sent to Sentry, the names and values of local variables that were set when the errors occurred are sent at the same time.
+
+You can stop sending local variables to Sentry by setting `include_local_variables=False` in the `sentry_sdk.init()` call.
+
+## SQL Queries
+
+While SQL queries are sent to Sentry, neither the full SQL query (`UPDATE app_user SET password='supersecret' WHERE id=1;`), nor the values of its parameters will ever be sent. A parameterized version of the query (`UPDATE app_user SET password='%s' WHERE id=%s;`) is sent instead.
+
```

Comment:
not vailable in .net

Reply:
@vaind I've left this in there for the time being as I think it's relevant for the `Sentry.DiagnosticSource` and `Sentry.EntityFramework` integrations. 

[Here](https://github.com/getsentry/sentry-dotnet/blob/3e70c3126515c87b1851b6a9a7afd99ffbb9aeb7/test/Sentry.DiagnosticSource.Tests/SentrySqlListenerTests.cs#L243), for example, we record the command query in a span description.

We can also capture this information via our OpenTelemetry integration. I got this from an app that calls out to Postgres, for example:
![image](https://github.com/user-attachments/assets/0d57a4c9-6ad7-4e31-8bf6-063f79b50b98)